transformer_engine/pytorch/attention.py

```diff
@@ -1391,12 +1391,12 @@ def attn_forward_func_with_cp(
     assert (qkv_format != 'thd' or \
             not use_fused_attention or \
             attn_mask_type in ["padding", "padding_causal"]
-        ), f"""Context parallelism is not supported for {attn_mask_type} mask type and \
-                {qkv_format} format with {"FusedAttention" if use_fused_attention else \
-                "FlashAttention"}!"""
+        ), f"""Context parallelism is not supported for {attn_mask_type} mask type and
+               {qkv_format} format with {"FusedAttention" if use_fused_attention else
+               "FlashAttention"}!"""
```

Comment:
This might print out all the leading spaces in the second and third lines. Try this:
```
        ), f"""Context parallelism is not supported for {attn_mask_type} mask type and """
        f"""{qkv_format} format with {"FusedAttention" if use_fused_attention else"""
        f""" "FlashAttention"}!"""
```

Reply:
Done